system-contracts/bootloader/bootloader.yul

```diff
@@ -4012,6 +4190,8 @@ object "Bootloader" {
             mstore(PRIORITY_TXS_L1_DATA_BEGIN_BYTE(), EMPTY_STRING_KECCAK())
             mstore(add(PRIORITY_TXS_L1_DATA_BEGIN_BYTE(), 32), 0)
 
+            setMessageRoots(0)
```

Comment:
we will already call it for txId 0 inside `processTx` function, maybe makes sense to delete it from here?